### PR TITLE
Update test_numeric_batchnorm.py

### PR DESCRIPTION
**Pull request by d-li14**
Fix `nn.BatchNorm2d` API incompatible with 2d tensor, tested with PyTorch version 0.4

### CODE REVIEW

**Review by vacancy — APPROVED** [review]
Thanks for the finding!
